src/fixtures/beacons.fixture.ts

```diff
@@ -1,25 +1,25 @@
 import faker from "faker";
 import { BeaconStatuses, BeaconTypes, IBeacon } from "../entities/IBeacon";
-import { testEmergencyContacts } from "./testEmergencyContacts";
-import { testOwners } from "./testOwner";
-import { testUses } from "./testUses";
+import { emergencyContactsFixture } from "./emergencyContacts.fixture";
+import { testOwners } from "./owner.fixture";
+import { usesFixture } from "./uses.fixture";
 
-export const testSingleBeacon: IBeacon = {
+export const beaconFixture: IBeacon = {
   id: "f48e8212-2e10-4154-95c7-bdfd061bcfd2",
   hexId: "1D0EA08C52FFBFF",
   type: BeaconTypes.Epirb,
   protocolCode: "AX7098",
   registeredDate: "2018-06-08T00:00",
   status: BeaconStatuses.New,
-  manufacturer: "OceanSignal",
+  manufacturer: "Ocean Signal",
   model: "Excelsior",
-  manufacturerSerialNumber: "123ABC",
+  manufacturerSerialNumber: "1407312904",
   chkCode: "456QWE",
   batteryExpiryDate: "2020-02-01T00:00",
   lastServicedDate: "2020-02-01T00:00",
-  uses: testUses,
+  uses: usesFixture,
   owners: testOwners,
-  emergencyContacts: testEmergencyContacts,
+  emergencyContacts: emergencyContactsFixture,
```

Comment:
Make properties in the fixtures folder consistent.  I.e. `singleBeaconResponseFixture` (the API response) is for the same beacon as `beaconFixture` (the modelled entity).